storybook/stories/core/components/PopOver/PopOver.stories.js

```diff
@@ -5,15 +5,23 @@ import PopOverStoryRightAligned from "../PopOver/PopOverRightAligned.story";
 export default {
     title: 'Popups/PopOver',
     component: PopOver,
+    argTypes: {
+        persistent: {
+            control: {type: 'boolean'},
+            table: {
+                type: {summary: 'Set persistent'},
+            },
+        },
+    }
 };
 
 const Template = (args) => ({
     setup() {
         return {args};
     },
-    components: {'oxd-input': PopOver},
-    template: '<oxd-input v-bind="args" />',
+    components: {'oxd-pop-over': PopOverStory},
+    template: '<div><oxd-pop-over v-bind="args" /></div>',
 });
 
-export const Default = () => PopOverStory;
+export const Default = Template.bind({});
```

Comment:
Better to introduce a story depicting the persistent behaviour

Reply:
Added new story